bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/channel/GroupAddressConfiguration.java

```diff
@@ -57,14 +57,28 @@ private GroupAddressConfiguration(@Nullable String dpt, GroupAddress mainGA, Set
         return dpt;
     }
 
+    /**
+     * Returns the main GA, which is the GA to send commands to.
+     */
     public GroupAddress getMainGA() {
         return mainGA;
     }
 
+    /**
+     * Returns all GAs to listen to.
+     * This includes the main GA (unless disabled by '>'), and additional listening GAs
+     * (those after the "+" symbol).
+     */
     public Set<GroupAddress> getListenGAs() {
         return listenGAs;
     }
 
+    /**
+     * Returns all GAs to read from.
+     * Those GAs accept read requests to the KNX bus, i.e. they respond to a "GroupValueRead" with a
+     * "GroupValueResponse".
+     * The '&lt;' sign sets a GA as read GA.
```

Comment:
```suggestion
     * The '<' sign sets a GA as read GA.
```

Any reason that this does not work? This would be much more readable when just browsing code ...

Reply:
javadoc error, < indicates a tag. Escaping can be &lt; or {@code <}